pages/login/index.tsx

```diff
@@ -32,9 +33,11 @@ const LoginPage = createPage({
   getInitialProps: async () => {},
   isPublic: true,
 })(() => {
+  const { NEXT_PUBLIC_ENABLE_LOGIN, NEXT_PUBLIC_ENABLE_REGISTRATION } = getConfig();
   const { token } = useAuthContext();
 
-  return token ? ( // logged in, so it shouldn't give you a login page
+  return token || // logged in, so it shouldn't give you a login page
+    !(NEXT_PUBLIC_ENABLE_LOGIN || NEXT_PUBLIC_ENABLE_REGISTRATION) ? (
     (Router.push({
       pathname: getInternalLink({ path: ROOT_PATH }),
     }),
```

Comment:
redirect to root instead of login, if user attempts a restricted feature while both are disabled.